### PR TITLE
Check shipped apps when running from cli

### PR DESCRIPTION
**Pull request by VicDeo**
## Description
`OC\IntegrityCheck\Checker::runInstanceVerification` checks core and shipped apps
`OC\IntegrityCheck\Checker::verifyCoreSignature` checks core only

## Related Issue
#26463

## Motivation and Context
Better integrity check from CLI

## How Has This Been Tested?
Current behavior:
1. modify a shipped app
2. check integrity from CLI `php occ integrity:check-core`
3. Nothing is reported
4. Login into admin area
5. Nothing is reported in WebUI
6. Modify core file
7. check integrity from CLI `php occ integrity:check-core`
8. Only core file is reported
9. Login into admin area
10. Both shipped app and  core file are reported

This PR:
Both shipped app and  core file are reported when running an integrity check from CLI

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by VicDeo** [comment]
@PVince81 @DeepDiver1975

File: core/Command/Integrity/CheckCore.php [hunk 1]
```diff
@@ -57,7 +57,8 @@ protected function configure() {
 	 * {@inheritdoc }
 	 */
 	protected function execute(InputInterface $input, OutputInterface $output) {
-		$result = $this->checker->verifyCoreSignature();
+		$this->checker->runInstanceVerification();
+		$result = $this->checker->getResults();
```
Comment (PVince81) [inline core/Command/Integrity/CheckCore.php]: what if someone really only wants to check core and not the shipped apps ? should we add a CLI option ?

not sure if this is really a use case, your change might be the most common

**Comment by PVince81** [comment]
:+1:

**Comment by PVince81** [comment]
Hmm, now thinking of it, this is a change of behavior. So it's ok for master but I don't think we should backport it.

**Comment by PVince81** [comment]
Rebased

**Pull request merged**